jena-arq/src/main/java/org/apache/jena/sparql/algebra/OpVisitor.java

```diff
@@ -23,53 +23,53 @@
 public interface OpVisitor
 {
     // Op0
-    public void visit(OpBGP opBGP) ;
-    public void visit(OpQuadPattern quadPattern) ;
-    public void visit(OpQuadBlock quadBlock) ;
-    public void visit(OpTriple opTriple) ;
-    public void visit(OpQuad opQuad) ;
-    public void visit(OpPath opPath) ;
-    public void visit(OpTable opTable) ;
-    public void visit(OpNull opNull) ;
+    default void visit(OpBGP opBGP) {}
```

Comment:
I don't know about this change. 

There are `OpVisitor` impls that do not use `OpVisitorBase` and that has the desirable (and intended) effect that a change in `OpVisitor` forces implementation. e.g. `OpVisitorByType` e.g. `ExecutionDispatch`

Having default methods masks this so it makes things less robust. There is rarely a general implementation of an `visit` that will work for everything.